electroncash_plugins/fusion/fusion.py

```diff
@@ -458,11 +459,16 @@ def run(self):
 
             self.status = ('complete', 'time_wait')
 
-            # wait up to a minute before unfreezing coins
+            # The server has told us the fusion is complete but we might not
+            # have seen the tx show up in our wallets. So we can't unfreeze
+            # coins or unreserve addresses yet, else they might be used in
+            # another fusion. Wait up to a minute for this to happen.
+            wallets = set(self.source_wallet_info.keys())
+            wallets.add(self.target_wallet)
```

Comment:
Heh. I guess you don't trust `|`?  Well `.add()` is a few microseconds faster so this is good...

😛